api/lib/infrastructure/repositories/user-repository.js

```diff
@@ -309,6 +323,22 @@ module.exports = {
       });
   },
 
+  async updateUserDetailsForAdministration(id, userAttributes) {
+
```

Comment:
Supprimer la ligne

Reply:
> Bon, si on se donnait comme objectif [ceux de Léo](https://blog.octo.com/en/seven-shades-of-git/), on y est pas, côté commit atomiques. My 2 cents.

Effectivement, en faisant du TDD in/out, j'ai eu du mal sur cette PR vers la fin. J'en ai parlé au point tech. Au début c'était atomique, jusqu'à ce que c'est devenu douloureux mes derniers rebase interactif.